code/campaign/src/main/java/com/adobe/marketing/mobile/campaign/FullScreenMessage.java

```diff
@@ -185,26 +191,19 @@ void showMessage() {
         final Map<String, String> cachedResourcesMap = createCachedResourcesMap();
 
         final FullScreenMessageUiListener fullScreenMessageUiListener = new FullScreenMessageUiListener();
-        final MessageSettings messageSettings = new MessageSettings();
         // ACS fullscreen messages are displayed at 100% scale
-        messageSettings.setHeight(FILL_DEVICE_DISPLAY);
-        messageSettings.setWidth(FILL_DEVICE_DISPLAY);
-        messageSettings.setParent(this);
-        messageSettings.setVerticalAlign(MessageSettings.MessageAlignment.TOP);
-        messageSettings.setHorizontalAlign(MessageSettings.MessageAlignment.CENTER);
-        messageSettings.setDisplayAnimation(MessageSettings.MessageAnimation.BOTTOM);
-        messageSettings.setDismissAnimation(MessageSettings.MessageAnimation.BOTTOM);
-        messageSettings.setBackdropColor("#FFFFFF"); // html code for white
-        messageSettings.setBackdropOpacity(0.0f);
-        messageSettings.setUiTakeover(true);
-        final FullscreenMessage fullscreenMessage = uiService.createFullscreenMessage(htmlContent,
-                fullScreenMessageUiListener, !cachedResourcesMap.isEmpty(), messageSettings);
-
-
-        if (fullscreenMessage != null) {
-            fullscreenMessage.setLocalAssetsMap(cachedResourcesMap);
-            fullscreenMessage.show();
-        }
+        final InAppMessageSettings messageSettings = new InAppMessageSettings.Builder()
+                .content(htmlContent)
+                .displayAnimation(InAppMessageSettings.MessageAnimation.BOTTOM)
```

Comment:
do ther vertical and horizontal alignment settings default to the desired settings (top and center)?

Reply:
Yes, but I will add it here just to be sure